images/inference-services/base/scripts/kube_to_fuseml_app.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
```

Comment:
I would make this file a library with functions that the predictor's run script can import.

Reply:
done